tests/python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest==6.0.1
+pandas==1.2.1
```

Comment:
wrong version of pandas => replace with pandas==0.23.4

Reply:
done in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/13/commits/053dfd6d93f7f8292cfcb96b52b0a5bea9b24aab